app/celery/scheduled_tasks.py

```diff
@@ -274,7 +274,7 @@ def check_for_services_with_high_failure_rates_or_sending_to_tv_numbers():
             )
             message += "service: {} failure rate: {},\n".format(service_dashboard, service.permanent_failure_rate)
     elif services_sending_to_tv_numbers:
-        message += "{} service(s) have sent over 500 sms messages to tv numbers in last 24 hours:\n".format(
+        message += "{} service(s) have sent high volumes of sms messages to tv numbers in last 24 hours:\n".format(
```

Comment:
Would encourage 'high volumes' to be a bit more explicit about what number we consider high volumes to be. For example, `have sent over 50,000 sms messages` or whatever limit we choose